pkgs/by-name/ma/mathemagix/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  stdenv,
+  lib,
+  fetchsvn,
+  readline,
+  ncurses,
+  bison,
+  libtool,
+  gmp,
+  mpfr,
+}:
+
+stdenv.mkDerivation {
+  pname = "mathemagix";
+  version = "11126";
+
+  src = fetchsvn {
+    url = "https://subversion.renater.fr/anonscm/svn/mmx/";
+    rev = 11126;
+    hash = "sha256-AFnYd5oFg/wgaHPjfZmqXNljEpoFW4h6f3UG+KZauEs=";
+  };
+
+  nativeBuildInputs = [
+    readline
+    ncurses
+    bison
+    libtool
+    gmp
+    mpfr
+  ];
+
+  preConfigure = ''
+    export HOME="$PWD"
+  '';
+
+  configureFlags = [ "--prefix=${placeholder "out"}" ];
```

Comment:
Already the default? See https://nixos.org/manual/nixpkgs/unstable/#variables-controlling-the-configure-phase